server/src/lib/model/statistics.ts

```diff
@@ -0,0 +1,239 @@
+  tableName: TableNames,
+  options?: QueryOptions
+): Promise<StatisticsCount[]> => {
+  const filter = options?.filter;
+  const conditional = filter && FILTERS[filter];
+
+  const [rows] = await db.query(`
+    SELECT
+      MAX(DATE_FORMAT(created_at, "%Y-%c-%d")) as date,
+      COUNT(created_at) as total_count
+    FROM
+      ${tableName} d
+    WHERE
+      created_at > now() - INTERVAL 12 MONTH
+      ${conditional ? 'AND ' + conditional : ''}
+    GROUP BY
+      DATE_FORMAT(created_at, "%Y-%c")
+    ORDER BY created_at DESC;
+  `);
+  return rows;
+};
+
+const getUniqueMonthlyContributions = async (
+  tableName: TableNames,
+  options: QueryOptions
+): Promise<StatisticsCount[]> => {
+  const { groupByColumn } = options;
+  const [rows] = await db.query(`
+  SELECT
+      MAX(DATE_FORMAT(created_at, "%Y-%c-%d")) as date,
+      COUNT(created_at) as total_count
+   FROM
+    (
+      SELECT * 
+      FROM ${tableName} d GROUP BY ${groupByColumn}
+    ) d
+    WHERE
+      created_at > now() - INTERVAL 12 MONTH
+    GROUP BY
+      DATE_FORMAT(created_at, "%Y-%c")
+    ORDER BY created_at DESC;
+  `);
+  return rows;
+};
+
+const getMonthlyDuplicateSentences = async (): Promise<StatisticsCount[]> => {
+  const [rows] = await db.query(`
+    SELECT
+      MAX(DATE_FORMAT(created_at, "%Y-%c-%d")) as date,
+      COUNT(created_at) as total_count
+    FROM
+      (
+      SELECT
+        created_at,
+        count(1) as sentenceCount
+      FROM
+        clips c
+      GROUP BY
+        original_sentence_id
+      HAVING
+        sentenceCount > 1
+        ) d
+    WHERE
+      created_at > now() - INTERVAL 12 MONTH
+    GROUP BY
+      DATE_FORMAT(created_at, "%Y-%c")
+    ORDER BY
+      created_at DESC;
+  `);
+  return rows;
+};
+
+const getTotalDuplicateSentences = async (): Promise<StatisticsCount> => {
+  const [[rows]] = await db.query(`
+    SELECT
+      SUM(d.total_count) as total_count
+    FROM
+      (
+      SELECT
+        count(1) as total_count
+      FROM
+        clips c
+      GROUP BY
+        original_sentence_id
+      HAVING
+        total_count > 1) d
+  `);
+  return rows;
+};
+
+const getUniqueSpeakersTotal = async (): Promise<StatisticsCount> => {
+  const [[rows]] = await db.query(`
+  SELECT
+	  count(d.total_count) as total_count
+  FROM
+    (
+    SELECT
+      count(1) as total_count
+    FROM
+      clips c
+    GROUP BY
+      client_id) d
+    `);
+  return rows;
+};
+const getTotal = async (
+  tableName: TableNames,
+  options?: QueryOptions
+): Promise<StatisticsCount> => {
+  const filter = options?.filter;
+  const conditional = filter && FILTERS[filter];
+
+  const [[rows]] = await db.query(`
+    SELECT
+    count(1) as total_count
+    FROM
+      ${tableName} d
+    ${conditional ? 'WHERE ' + conditional : ''}
+  `);
+  return rows;
+};
+
+export const getStatistics = lazyCache(
+  'get-stats',
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
It seems the `_filter` parameter is not used anywhere in this function. So is it ok to remove it and the eslint comment?